<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMolPhot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="centralLayout">
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLoadConfFromFile"/>
    <addaction name="actionSaveConfToFile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="parametersDockWidget">
   <property name="windowTitle">
    <string>Parameters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="parametersDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="ParameterTree" name="parametersTree" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionLoadConfFromFile"/>
   <addaction name="actionSaveConfToFile"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources/resource.qrc">
     <normaloff>:/48x48/48x48/Open_48x48.png</normaloff>:/48x48/48x48/Open_48x48.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources/resource.qrc">
     <normaloff>:/48x48/48x48/Save_48x48.png</normaloff>:/48x48/48x48/Save_48x48.png</iconset>
   </property>
   <property name="text">
    <string>Save state</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources/resource.qrc">
     <normaloff>:/48x48/48x48/Log Out_48x48.png</normaloff>:/48x48/48x48/Log Out_48x48.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoadConfFromFile">
   <property name="icon">
    <iconset resource="resources/resource.qrc">
     <normaloff>:/48x48/48x48/Stock Index Down_48x48.png</normaloff>:/48x48/48x48/Stock Index Down_48x48.png</iconset>
   </property>
   <property name="text">
    <string>Load config</string>
   </property>
  </action>
  <action name="actionSaveConfToFile">
   <property name="icon">
    <iconset resource="resources/resource.qrc">
     <normaloff>:/48x48/48x48/Stock Index Up_48x48.png</normaloff>:/48x48/48x48/Stock Index Up_48x48.png</iconset>
   </property>
   <property name="text">
    <string>Save config</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParameterTree</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.parametertree</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
